### PR TITLE
changes in dependencies.rst to remove cython related issue1374

### PR DESCRIPTION
**Pull request by skc222**

### CODE REVIEW

File: DEPENDENCIES.rst [hunk 1]
```diff
@@ -165,6 +165,13 @@ To determine which version of Python is already installed on your computer, run:
 
    python -V
 
+Cython Note
+^^^^^^^^^^
```
Comment (scopatz) [inline DEPENDENCIES.rst]: Underlines in reStructured test must be as long or longer than the title.

File: DEPENDENCIES.rst [hunk 2]
```diff
@@ -165,6 +165,13 @@ To determine which version of Python is already installed on your computer, run:
...
+If you get an error related to an old Cython version, then this may be 
+because the Debian stable version of Cython is 0.23.4. However, 
+Cyclus requires .25+. To install the latest cython version, please
```
Comment (scopatz) [inline DEPENDENCIES.rst]: Please put the traditional 0 in front of `0.25+`. Thanks!

**Comment by scopatz** [comment]
Thanks for putting this in @skc222, a couple of small issues to fix up.

**Comment by skc222** [comment]
Done! and thanks @scopatz  for reviewing the changes.

**Comment by scopatz** [comment]
LGTM, just waiting for CI.

**Comment by bam241** [comment]
@skc222 @scopatz thx !

I'll update the fuel cycle.org to have to change included. (we automatically grab them from here!)

**Comment by bam241** [comment]
we actually grab this from `master` so I can only force it, shall I do it ?

**Comment by bam241** [comment]
@scopatz @katyhuff @gonuke 

when we build `fuelcycle.org`, it now grabs some documentation files from `cyclus/master`
(`INSTALL.rst` and `DEPENDENCIES.rst`)

shall me add this to master, to be able to use the normal hook for `fuelcycle.org` ? (don't necessary need an release update)

We can also manually specify which branch to hook, but it will be overwritten at the next "non -careful" `fuelcycle.org` update....

**Comment by scopatz** [comment]
CEP3 says that we don't touch master except at a release.

**Comment by bam241** [comment]
shall we rethink documentation then ?

**Comment by scopatz** [comment]
I don't think it is a priority, but I won't say no to anyone who wants to put in the effort :)

**Comment by bam241** [comment]
I'll give it a think :)

**Comment by katyhuff** [comment]
Maybe it's time for a minor version release?

**Comment by scopatz** [comment]
Definitely overdue for a release :)

**Pull request merged**